pkg/tools/tools.go

```diff
@@ -58,6 +58,12 @@ func GetCurrentNS() (string, error) {
 	return "", fmt.Errorf("namespace is not found")
 }
 
+// GetCurrentPodNameFromHostname returns pod name a container is running in
+func GetCurrentPodNameFromHostname() (string, error) {
+	podName, err := os.Hostname()
```

Comment:
Please add a note here that the fact that the pod name is present in `/etc/hostname` is not fixed in k8s docs. Plus there can be a case where `someone` rewrites that file. However, we know that in the 99.999% of the cases this is true for now but may change in the future.